data/remnant missions.txt

```diff
@@ -814,6 +814,17 @@ mission "Remnant: Taely Intro Log"
 
 
 
+mission "Remnant: Tali to Taely"
```

Comment:
No instances of `"remnant tali"` even exist right now.

Reply:
Ah, yeah. That was added during the de-duplification stuff.